edflow/eval/pipeline.py

```diff
@@ -1194,7 +883,7 @@ def main():
 
     callbacks = cbargs2cbdict(args.callback)
 
-    standalone_eval_csv_file(args.csv, callbacks, additional_kwargs, args.other_config)
+    standalone_eval_meta_dset(args.csv, callbacks, additional_kwargs, args.other_config)
```

Comment:
update argparser, documentation and tests to reflect the change from csv to meta yaml and we're ready to merge